.gitignore

```diff
@@ -28,3 +28,6 @@ env.yml-original
 .env
 !configurations/test/env.yml
 scripts/*client.json
+
+# Vs code settings
+.vscode
```

Comment:
Super nit I'd add a trailing slash here to make it clear it's a folder!

Reply:
Super nit could be a great marvel character, you may want to trademark that

Reply:
Fixed in https://github.com/ibi-group/datatools-ui/pull/772/commits/da2ba045945c8b64d06d0c3f207b8d38b89759f2